external-crates/move/move-compiler/src/cfgir/translate.rs

```diff
@@ -256,8 +257,152 @@ fn script(context: &mut Context, hscript: H::Script) -> G::Script {
 // Functions
 //**************************************************************************************************
 
+fn constants(
```

Comment:
Maybe a small comment at least about what is going on here in terms of dependencies